### PR TITLE
USHIFT-351: Microshift: skip sig-devex tests

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
Ready for review.
/assign @ingvagabund

**Comment by pacevedom** [comment]
/hold
Need to get of microshift checks

File: test/extended/controller_manager/pull_secret.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
 		}
 	})
 
-	g.It("TestDockercfgTokenDeletedController", func() {
+	g.It("TestDockercfgTokenDeletedController [apigroup:image.openshift.io]", func() {
```
Comment (ingvagabund) [inline test/extended/controller_manager/pull_secret.go:132]: Where does the image apigroup requirement come from? `waitForServiceAccountPullSecret`?

Comment (pacevedom) [reply test/extended/controller_manager/pull_secret.go:132]: Yes, `waitForServiceAccountPullSecret` expects a secret to be provisioned in order to make use of the image API.

Comment (ingvagabund) [reply test/extended/controller_manager/pull_secret.go:132]: Is it possible to say by which component is the secret provisioned?

Comment (pacevedom) [reply test/extended/controller_manager/pull_secret.go:132]: That would be the openshift controller manager.

Comment (ingvagabund) [reply test/extended/controller_manager/pull_secret.go:132]: TestAutomaticCreationOfPullSecrets follows the same workflow with the default sa name. Checking for presence of image/cluster object.

Comment (pacevedom) [reply test/extended/controller_manager/pull_secret.go:132]: That test is updated too: https://github.com/openshift/origin/pull/27392/files#diff-e548b9ac6cbe0bee7cc02da7c4b6ced6eb44801339dfea05220e4285b7eec558R63
Or did I misunderstand you?

Comment (ingvagabund) [reply test/extended/controller_manager/pull_secret.go:132]: I was just justifying why adding image api group into `TestDockercfgTokenDeletedController` is required. The test itself does not give any indication why the image group is needed. Though, following `TestAutomaticCreationOfPullSecrets` gives some idea.

**Review by soltysh — APPROVED** [review]
/approve

File: test/extended/controller_manager/pull_secret.go (hunk 1 above)
Comment (pacevedom) [reply test/extended/controller_manager/pull_secret.go:132]: Oh ok, didnt follow you. Thanks!

**Pull request merged**